src/main/resources/templates/pdf/willLodgementDepositReceipt.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org" xmlns="http://www.w3.org/1999/html">
+<head>
+    <title th:remove="all" th:text="${title}">Grant of Probate</title>
```

Comment:
Think this title is used for Grant templates, might need to change this for Will Lodgement Receipt.